spark/v3.4/spark-extensions/src/test/java/org/apache/iceberg/spark/extensions/SparkRowLevelOperationsTestBase.java

```diff
@@ -95,21 +101,23 @@ public SparkRowLevelOperationsTestBase(
       String distributionMode,
       boolean fanoutEnabled,
       String branch,
-      PlanningMode planningMode) {
+      PlanningMode planningMode,
+      int formatVersion) {
     super(catalogName, implementation, config);
     this.fileFormat = fileFormat;
     this.vectorized = vectorized;
     this.distributionMode = distributionMode;
     this.fanoutEnabled = fanoutEnabled;
     this.branch = branch;
     this.planningMode = planningMode;
+    this.formatVersion = formatVersion;
```

Comment:
Note to reviewer, the 3.4 backport changes are slightly different here due to 3.5 tests using Junit5. In this PR the parameterization requires explicit passing of the formatVersion through all the constructors.